tools/cli/adminCommands.go

```diff
@@ -43,7 +43,7 @@ const (
 func getAdminServiceClient(c *cli.Context) adminserviceclient.Interface {
 	client, err := cBuilder.BuildAdminServiceClient(c)
 	if err != nil {
-		ExitIfError(err)
```

Comment:
Are we deprecating this method? ExitIfError
Then probably we should delete it if not being used anywhere.